rasa/core/training/story_reader/yaml_story_reader.py

```diff
@@ -156,6 +162,12 @@ def _validate_that_utterance_is_in_domain(self, utterance: UserUttered) -> None:
                 docs=DOCS_URL_STORIES,
             )
 
+    def _parse_user_message(self, step: Dict[Text, Any]) -> None:
+        message = step.get(KEY_STORY_USER_END_TO_END_MESSAGE, "")
+        self.current_step_builder.add_user_messages(
+            [UserUttered(message, {"name": None})]
+        )
+
```

Comment:
Are we planning to support `[value](entity_name)` entities? if so, we need to insert them into e2e text (`Book a [Chinese](cuisine) restaurant` -> `Book a Chinese restaurant`)

Reply:
Mhm, I'm actually a big fan of dumping that and separate keys for that in yml, too. Anyway, for now I think we should support that 👍 Will add it

Reply:
done